tasks/centos.yml

```diff
@@ -0,0 +1,13 @@
+- name: "Define proper service name"
```

Comment:
Yes that's ok.
No, except including another file instead of defining the variable, you can't do anything else. But it adds more complexity.